src/components/App.jsx

```diff
@@ -45,22 +45,25 @@ class App extends React.Component {
     //actions if necessary.
     apiClient.beforeEveryRequest = this.checkIfLoggedInUserIsStillLoggedIn.bind(this);
   }
-  
+
   checkIfLoggedInUserIsStillLoggedIn() {
     const props = this.props;
-    
+
     return oauth.checkBearerToken()
       .then((token) => {
         //If the App thinks you're logged in, but the token says otherwise, deploy emergency measures.
-        if (props.initialised && props.user && !token) {          
+        if (props.initialised && props.user && !token) {
           this.props.dispatch(emergencySaveWorkInProgress());
           this.props.dispatch(toggleDialog(<DialogOfFailure />, false, true));
           return Promise.reject(new Error('User is supposed to be logged in, but token has expired.'));
           //The intent is that if the user is supposed to be logged in but
           //isn't, the whole request (that comes after .beforeEveryRequest)
           //should not continue.
-        }      
+        }
         return;
+      })
+      .catch((err) => {
```

Comment:
Wait, the line below literally says `//Note: do NOT add a catch() here, or else Promise.reject() above won't stop the API request.` - are we sure we want to do this?

Reply:
I need to double check this, but if we add `.catch()` here, then if any other API call (e.g. fetch workflow, submit Classification) fails due to a bad bearer token, then those API calls' respective `.catch()`es won't trigger.

Reply:
Yep. The error here means that your login session has expired and couldn't be renewed (probably because the browser or a privacy extension blocks zooniverse.org while you're on antislaverymanuscripts.org.)

I'd guess that catching that error means you'll lose your classification work?

Reply:
I failed to see that comment below, so I'm glad you're pointing it out to me now. That error has ceased being reported from Rollbar, so I'm ok removing the `catch` here if we're worried about adverse effects.